README.md

```diff
@@ -13,3 +13,218 @@ This is an NWB extension for storing metadata of devices used in optical experim
 
 ---
 This extension was created using [ndx-template](https://github.com/nwb-extensions/ndx-template).
+
+## Entity relationship diagrams
+
+#### Indicator and Effector
+
+```mermaid
+%%{init: {'theme': 'base', 'themeVariables': {'primaryColor': '#ffffff', "primaryBorderColor': '#144E73', 'lineColor': '#D96F32'}}}%%
+classDiagram
+    direction BT
+    class Indicator{
+        <<NWBContainer>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        label : text
+        description : text, optional
+        manufacturer : text, optional
+        injection_location : text, optional
```

Comment:
Not that there was too much conclusion from the LBNL meeting, but perhaps `injection_brain_region`?